.github/workflows/build.yml

```diff
@@ -13,8 +13,50 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.15.x
+          go-version: 1.16
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Build
         run: make build
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build docs
+        run: make docs
+      - run: git add -A
+      - run: git diff --no-ext-diff --exit-code
+        name: Check if there are changes
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
```

Comment:
We have that setup in CLI using golangci. Best to use the same config:

https://github.com/redhat-developer/app-services-cli/blob/develop/.golangci.yaml